### PR TITLE
feat(landing): initiate page with basic design

### LINKED ISSUE #37: Mock landing screen from design
## Description

This ticket addresses the product definition and explanation to be added to the CC landing screen

Also, for the AI Dev/Co a shortened version should be available so they don't lose focus but know what it's up to.

### PR DESCRIPTION
**Pull request by juanmanso**
## Description

As part of the productisation process, we need to have a more appealing and cool looking landing page.

For that we based our design on the NVM's app page. To do so, we created a bypass on the nextjs component selection to go through this new page instead of the previous one. In future PRs we shall restructure the whole app and cleanup the Scaffold-eth template.

Now, we just have a page filled with little components that could later be refactored into utility components but at least it gives the broad idea of the intended design to keep iterating.

Things developed to make this possible:
- New input component
  - Doesn't completety follow the design now but at it's a start so we iterate over it but have the base layout.
- Enable Text components to be styled and act as buttons
- Add svg and png assets

## Related issues

- Closes #37 

## Screenshots

| _Sample_ |
| :---: |
| image  |

### CODE REVIEW

File: packages/nextjs/pages/_app.tsx [hunk 1]
```diff
@@ -18,12 +19,13 @@ import { wagmiConfig } from "~~/services/web3/wagmiConfig";
 import { appChains } from "~~/services/web3/wagmiConnectors";
 import "~~/styles/globals.css";
 
-const ScaffoldEthApp = ({ Component, pageProps }: AppProps) => {
+const ScaffoldEthApp = ({ Component, pageProps, router }: AppProps) => {
```
Comment (fabriziogianni7) [inline packages/nextjs/pages/_app.tsx:22]: should we rename that later on? ScaffoldEthApp --> App

File: packages/nextjs/pages/landing.tsx [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+              Log In
+            </Button>
+          );
+        }}
+      </ConnectButton.Custom>
+    </div>
+  );
+};
+
+const Title = () => {
```
Comment (fabriziogianni7) [inline packages/nextjs/pages/landing.tsx:49]: should you put this in a separate file?

File: packages/nextjs/pages/landing.tsx
Comment (fabriziogianni7) [inline packages/nextjs/pages/landing.tsx:1]: should we put all of these in separate files?

File: packages/nextjs/pages/_app.tsx (hunk 1 above)
Comment (juanmanso) [reply packages/nextjs/pages/_app.tsx:22]: Yes we should, I'm actually in the process of doing it. Thanks for the reminder tho 😄

File: packages/nextjs/pages/landing.tsx
Comment (juanmanso) [reply packages/nextjs/pages/landing.tsx:1]: Yes but I'm working on it 😄

File: packages/nextjs/pages/landing.tsx [hunk 3]
```diff
@@ -241,11 +241,13 @@ const Footer = () => {
...
+const Landing = () => {
   return (
     <div className="flex flex-col" style={{ flex: 4, height: "100%" }}>
       <Body />
       <Footer />
     </div>
   );
 };
+
+export default Landing;
```
Comment (juanmanso) [inline packages/nextjs/pages/landing.tsx:253]: No need for default export. If this is throwing some errors, then let's leave it like this for now but will be changed later on

**Review by juanmanso — COMMENTED** [review]
Thanks for the review, leaving answering some comments :)

**Pull request merged**